claat/parser/md/parse.go

```diff
@@ -33,9 +33,9 @@ import (
 	"golang.org/x/net/html"
 	"golang.org/x/net/html/atom"
 
-	"github.com/googlecodelabs/tools/claat/parser"
 	"github.com/googlecodelabs/tools/claat/types"
 	"github.com/russross/blackfriday"
+	"github.com/googlecodelabs/tools/claat/parser"
```

Comment:
This doesn't look right. Feels like gofmt wasn't run on this file?
I mean, the imports seem out of order.